src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -118,19 +178,54 @@ public ReturnValueConfiguration(RuleBuilder parentConfiguration)
 
             public ICallMatcher Matcher => this.ParentConfiguration.Matcher;
 
+            public IReturnValueConfiguration<TMember> Then
+            {
+                get
+                {
+                    var newConfiguration = (RuleBuilder)this.ParentConfiguration.Then;

```

Comment:
(btw, "Sure" wasn't referring to "Am I just being silly", in case that wasn't clear :wink:)